app/views/welcome/index.html.erb

```diff
@@ -1,5 +1,200 @@
-<div class="text-center">
-  <h1 class="page-header"><%= t 'appname' %></h1>
-  <img class="img-responsive" src="https://bluefin-live-assets.s3.amazonaws.com/media/filer_public_thumbnails/filer_public/30/dc/30dcb5b3-815d-4e4c-af3e-d2823050911c/sports_montage_-_chosen_concept_v9_carouselv3_rgb.jpg__1600x580_q85_crop_upscale.jpg"/>
-  <p class="lead"><%= t '.lead' %></p>
+<!-- Add css file manually, since it does not get linked automatically as it should -->
+<%= stylesheet_link_tag 'welcome' %>
+
+<div class="section darkblue-soccer">
+  <div class="container">
+    <div class="row">
+      <div class="col s12">
+        <h1 class="white-text center"><%= t('welcome.index.lead') %></h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col s10 offset-s1">
+        <p class="white-text center" id="header-description"><%= t('welcome.index.lead-description') %></p>
+      </div>
+    </div>
+    <div id="register-row" class="row">
+      <div class="col s12 center">
+        <% if not user_signed_in? %>
```

Comment:
According to the User Story, the button should still be present when logged in and take the user to his dashboard.